From 3245b0f9ed2085f6167068409fb344166093808c Mon Sep 17 00:00:00 2001
From: Cristian Stoica <cristian.stoica@nxp.com>
Date: Tue, 23 May 2017 15:50:40 +0300
Subject: [PATCH 6/9] move structure definition to cryptodev_int.h

This is necessary for the rsa patch and makes this data structure
visible to kernel_crypt_pkop structure which will be defined in
cryptodev_int.h as well.

Signed-off-by: Cristian Stoica <cristian.stoica@nxp.com>
---
 cryptlib.h      | 6 ------
 cryptodev_int.h | 5 +++++
 2 files changed, 5 insertions(+), 6 deletions(-)

diff --git a/cryptlib.h b/cryptlib.h
index 8e8aa71..48fe9bd 100644
--- a/cryptlib.h
+++ b/cryptlib.h
@@ -2,12 +2,6 @@
 # define CRYPTLIB_H
 
 #include <linux/version.h>
-
-struct cryptodev_result {
-	struct completion completion;
-	int err;
-};
-
 #include "cipherapi.h"
 
 struct cipher_data {
diff --git a/cryptodev_int.h b/cryptodev_int.h
index d7660fa..c1879fd 100644
--- a/cryptodev_int.h
+++ b/cryptodev_int.h
@@ -35,6 +35,11 @@
 #define ddebug(level, format, a...) dprintk(level, KERN_DEBUG, format, ##a)
 
 
+struct cryptodev_result {
+	struct completion completion;
+	int err;
+};
+
 extern int cryptodev_verbosity;
 
 struct fcrypt {
-- 
2.7.4

